### PR TITLE
Backport keycloak 8.0.1 theme changes

### PR DESCRIPTION
**Pull request by vanrar68**

### CODE REVIEW

File: keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js [hunk 1]
```diff
@@ -970,6 +1014,12 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         "request_uri only"
     ];
 
+	$scope.changePkceCodeChallengeMethodOptions = [
```
Comment (fperot74) [inline keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js:1017]: Please fix the indentation

File: keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js [hunk 2]
```diff
@@ -1150,6 +1200,8 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
 
         $scope.accessTokenSignedResponseAlg = $scope.client.attributes['access.token.signed.response.alg'];
         $scope.idTokenSignedResponseAlg = $scope.client.attributes['id.token.signed.response.alg'];
+		$scope.idTokenEncryptedResponseAlg = $scope.client.attributes['id.token.encrypted.response.alg'];
```
Comment (fperot74) [inline keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js:1203]: Please fix the indentation

File: keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js [hunk 3]
```diff
@@ -1160,6 +1212,9 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         var attrVal3 = $scope.client.attributes['request.object.required'];
         $scope.requestObjectRequired = attrVal3==null ? 'not required' : attrVal3;
 
+		var attrVal4 = $scope.client.attributes['pkce.code.challenge.method'];
```
Comment (fperot74) [inline keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js:1215]: Please fix the indentation

File: keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js [hunk 4]
```diff
@@ -1271,6 +1326,13 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         $scope.clientEdit.attributes['id.token.signed.response.alg'] = $scope.idTokenSignedResponseAlg;
     };
 
+	$scope.changeIdTokenEncryptedResponseAlg = function() {
```
Comment (fperot74) [inline keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js:1329]: Please fix the indentation

File: keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js [hunk 5]
```diff
@@ -1299,6 +1361,9 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         }
     };
 
+	$scope.changePkceCodeChallengeMethod = function() {
```
Comment (fperot74) [inline keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/js/controllers/clients.js:1364]: Please fix the indentation

**Comment by vanrar68** [comment]
Sorry, wrong git command. Will resubmit PR later if I can figure out why indentation is ok on my side and not on github side

**Pull request closed without merging**